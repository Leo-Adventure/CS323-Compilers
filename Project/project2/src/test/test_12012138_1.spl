struct rabbit
{
  int weight;
  float round;
};
int test_2_r07()
{
  struct rabbit r1;
  struct rabbit r2;
  struct rabbit r3 = r1 + r2;
  int r1_weight = r1.weight;
  return 0;
}