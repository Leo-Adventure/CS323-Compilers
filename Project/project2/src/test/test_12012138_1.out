Error type 7 at line 10: binary operation on non-number variables
Error type 5 at line 10: unmatching type on both sides of assignment